_episodes/01-introduction.md

```diff
@@ -116,19 +118,20 @@ the packaging of a complete software system:
 > addition to its own file system and has to get booted up in the same way 
 > a computer would. 
 > A container is considered a lightweight version of a virtual machine; 
-> underneath, the container is using the Linux 
-> kernel and simply has some flavor of Linux + the file system inside. 
+> underneath, the container is using the Linux kernel and simply has some 
+> flavor of Linux + the file system inside. 
 {: .callout}
 
-One final term: if the container is an alternative file system layer that you 
-can access and run from your computer, the **container image** is like a template 
-for that container. The container image has all the needed information to start 
+One final term: while the **container** is an alternative file system layer that you 
+can access and run from your computer, the **container image** is the 'recipe' or template
+for a container. The container image has all the needed information to start 
 up a running copy of the container. A running container tends to be transient 
-and can be started and shut down. The image is more long-lived, as a source file for the container. 
-You could think of the container image like a cookie cutter -- it 
-can be used to create multiple copies of the same shape (or container) 
-and is relatively unchanging, where cookies come and go. If you want a 
-different type of container (cookie) you need a different image (cookie cutter).
+and can be started and shut down. The image is more long-lived: as a source file 
+for the container. It can also serve as documentation for what is inside the container, 
+like the list of ingredients in a cookie recipe. The container image can be used to 
+create batches of the same cookie (i.e. multiple containers) and is relatively unchanging, 
+whereas the individual cookies come and go. If you want a different type of container 
+(cookie), then you need a different image (recipe).
```

Comment:
Just wondering if the use of the term `recipe` is right here? I would normally think of a Dockerfile as being a recipe and the image being the thing generated from that recipe?